Intersphinx (#92)
lyft/flytesnacks: cookbook/docs/plugins.rst

What are Flytekit [python] only plugins?
===========================================
Flytekit plugins are simple plugins that can be implemented purely in python, unit tested locally and allow extending Flytekit functionality. These plugins can be anything and for comparison can be thought of like Airflow Operators.
Data is automatically marshalled and unmarshalled into and out of the plugin and mostly users should implement :pyclass:`flytekit.annotated.python_function_task.PythonTask` API, defined in flytekit.
Data is automatically marshalled and unmarshalled into and out of the plugin and mostly users should implement :py:class:`flytekit.annotated.base_task.PythonTask` API, defined in flytekit.

> This tutorial will walk you through an example of how a plugin can be written.

Reviewer: ```suggestion
 This tutorial will illustrate how a plugin can be implemented with the help of an example.
```
